app/_common/shadcn/ui/accordion.tsx

```diff
@@ -2,22 +2,36 @@
 
 import * as React from "react";
 import { ChevronDown } from "lucide-react";
+import { VariantProps, cva } from "class-variance-authority";
 import * as AccordionPrimitive from "@radix-ui/react-accordion";
 
 import { cn } from "@/app/_common/shadcn/utils";
 
+const accordionVariants = cva(
+  "hover:neo-hover shadow-neo-thin px-5 rounded-md border-black",
+  {
+    variants: {
+      variant: {
+        default: "",
+        outline: "border",
+      },
+    },
+    defaultVariants: {
+      variant: "default",
+    },
+  },
+);
+
 const Accordion = AccordionPrimitive.Root;
 
 const AccordionItem = React.forwardRef<
   React.ElementRef<typeof AccordionPrimitive.Item>,
-  React.ComponentPropsWithoutRef<typeof AccordionPrimitive.Item>
->(({ className, ...props }, ref) => (
+  React.ComponentPropsWithoutRef<typeof AccordionPrimitive.Item> &
+    VariantProps<typeof accordionVariants>
+>(({ className, variant, ...props }, ref) => (
   <AccordionPrimitive.Item
     ref={ref}
-    className={cn(
-      "hover:neo-hover rounded-md border border-black shadow-neo-thin px-5",
-      className,
-    )}
+    className={cn(className, accordionVariants({ variant }))}
```

Comment:
저도 이거 항상 헷갈리는데요.
컴포넌트 커스텀 자율성을 위해서 넣은 className이 순서가 앞으로 오면 의도한대로 적용이 안될 수 있어요. 
순서가 저도 헷갈려서...이렇게 수정 제안드려요.
```suggestion
    className={cn(accordionVariants({ variant, className }))}
```

+) 확인해보니까 badge.ts의 className은 순서가 뒤에 가있더라구요 이것도 혹시 모르니 마찬가지로 Variants 안으로 옮겨주세요~

Reply:
tailwind css class 적용 순위에 관해서 제가 [카카오 블로그](https://fe-developers.kakaoent.com/2022/220303-tailwind-tips/)  읽어본건데
참고하시라구 두고 갈게요!
![스크린샷 2024-03-14 오후 7 04 31](https://github.com/Open-Eye-Im-Developer/MoGakGo-FE/assets/114740795/04da74a6-1017-41e7-8ab6-0457ee829b99)

Reply:
수정했습니다!